package/MDAnalysis/analysis/hbonds/wbridge_analysis.py

```diff
@@ -314,13 +494,42 @@ class WaterBridgeAnalysis(HydrogenBondAnalysis):
 
     .. versionadded:: 0.17.0
     """
+    # use tuple(set()) here so that one can just copy&paste names from the
+    # table; set() takes care for removing duplicates. At the end the
+    # DEFAULT_DONORS and DEFAULT_ACCEPTORS should simply be tuples.
+
+    #: default heavy atom names whose hydrogens are treated as *donors*
```

Comment:
I dislike the code/data duplication between here and HydrogenBondAnalysis.

How about creating a mixin-class for both that contains these data?

Reply:
As you have commented that we will remove the old hba.